ydkgen/printer/go/function_printer.py

```diff
@@ -32,6 +32,20 @@ def __init__(self, ctx, clazz, leafs=None, children=None):
             identifier = "self"
         self.class_alias = identifier
 
+    def quick_print(self, name, args='', return_type=' ', stmt='', return_stmt=''):
```

Comment:
Could a better name for this be `print_function` instead of `quick_print` (which is a little vague)?

Reply:
I see your point, but the file is named 'function_printer' and the class is named 'FunctionPrinter', so print_function would be more vague.

A quick explanation of the class:
The FunctionPrinter class is meant to be inherited by other classes dedicated to printing complicated functions such as GetChildrenPrinter and GetSegmenetPathPrinter. Classes that inherit FunctionPrinter will define how to printer the function header and body. For simple getter/setter functions, it was more efficient to forgo defining a separate class and have helper functions to take in a few args and quickly print out a function. 

I can add some documentation so that the use will be clearer.